.changeset/five-doors-love.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
+---
+
+Support AVIF input assets
```

Comment:
Given this is a minor can you make this a little more flowery... almost as if it were going into a blog post.

Reply:
Done!